src/app/shared/paging/episode-page.component.ts

```diff
@@ -20,17 +22,36 @@ export class EpisodePageComponent implements OnChanges {
   @Input() totalPages;
   @Output() pageChange = new EventEmitter<number>();
   pages: number[];
-  showNumPages = 10;
+  showNumPages = 5;
```

Comment:
there's a test you'll need to update for this one -- currently failing 
`should only show up to showNumPages at a time` => 	`Expected 5 to equal 10.`

Reply:
good catch. seems like the test should be?
```diff
-    expect(comp.pager.pagesEnd).toEqual(10);
+    expect(comp.pager.pagesEnd).toEqual(comp.pager.showNumPages);
```
to prevent that from happening if the config ever changes. Do we want to test config specifics, or just that the config is respected in the code execution?

Reply:
testing that the config is respected is my preference